glossary/it/human-readable/index.md

```diff
@@ -0,0 +1,8 @@
+---
+section: terms
+lang: it
+title: Human Readable
+---
+
+I dati sono in formato che non sono facilmente leggibili da una persona. Alcuni formati human-readable, come il [PDF][/glossary/it/terms/pdf/), non sono {macchine-readable} e 
+Data in a format that can be conveniently read by a human. Some human-readable formats, such as [PDF](/glossary/en/terms/pdf/), are not {machine-readable} in quanto non sono [dati strutturati](/glossary/it/terms/structured-data/), e.s. la rappresentazione di dati su un disco non rappresenta la reale relazione fra i dati presenti.
```

Comment:
@napo - before I merge, do you want to change the link here (PDF) to the italian one?